test/colony-network-mining.js

```diff
@@ -1758,6 +1681,7 @@ contract("ColonyNetworkMining", accounts => {
       await giveUserCLNYTokensAndStake(colonyNetwork, OTHER_ACCOUNT2, DEFAULT_STAKE);
       await giveUserCLNYTokensAndStake(colonyNetwork, OTHER_ACCOUNT3, DEFAULT_STAKE);
 
+      await fundColonyWithTokens(metaColony, clny, INITIAL_FUNDING.muln(4));
```

Comment:
Why was this additional funding necessary?

Reply:
Not sure, it showed up when I rebased onto an earlier PR.

Reply:
Yup, I remember now -- there was a funding line added to the `beforeEach` block for this suite of tests, so this isn't necessary. It's a tad opaque for my taste but so be it.

Reply:
It might look better once I split them off and they all get their individual `beforeEach`s

Reply:
:D :D :D